keras_nlp/layers/preprocessing/masked_lm_mask_generator.py

```diff
@@ -35,7 +38,9 @@ class MaskedLMMaskGenerator(keras.layers.Layer):
     selected token, it has a chance (configurable) to be replaced by
     "mask token" or random token, or stay unchanged.
 
-    Users should use this layer with `tf.data` to generate masks.
```

Comment:
Is this no longer true? It seems like `tf.data` is probably still the expected path here

Reply:
It is! I will just leave in for now.

All our preprocessing and tokenizing layers would really expect `tf.data` as the best practice path. Just none of our other docstrings are calling it out as such.